app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, x: int, y: int) -> None:
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, vector_2: Vector) -> Vector:
+        return Vector(self.x + vector_2.x, self.y + vector_2.y)
+
+    def __sub__(self, vector_2: Vector) -> Vector:
+        return Vector(self.x - vector_2.x, self.y - vector_2.y)
+
+    def __mul__(self, multiplier: Vector | int) -> float | Vector:
+        if isinstance(multiplier, Vector):
+            return self.x * multiplier.x + self.y * multiplier.y
+
+        elif isinstance(multiplier, int) or isinstance(multiplier, float):
+            return Vector(self.x * multiplier, self.y * multiplier)
+
+    @classmethod
+    def create_vector_by_two_points(cls, start_point: tuple,
+                                    end_point: tuple) -> Vector:
+        return cls(end_point[0] - start_point[0],
+                   end_point[1] - start_point[1])
+
+    def get_length(self, vector: Vector) -> float:
+        return pow((pow(vector.x, 2) + pow(vector.y, 2)), 1 / 2)
```

Comment:
Rzuc okiem na moduł `math`. Ma wbudowane funckje matematyczne takie jak `sqrt`

```suggestion
        return sqrt(self.x ** 2 + self.y ** 2)
```